assets/chat/js/messages/ChatUserMessage.js

```diff
@@ -108,4 +116,11 @@ export default class ChatUserMessage extends ChatMessage {
     this.ui.classList.toggle('msg-own', isOwn);
     this.isown = isOwn;
   }
+
+  setWatching(user) {
+    this.ui.classList.toggle(
+      'watching-same',
+      user.watching && user.equalWatching(this.watching),
```

Comment:
If the logged in user switches from watching something to watching nothing while watching focus is on, would this dim all the messages?